modules/apps/commerce/commerce-cart-content-web/src/main/resources/META-INF/resources/cart/view.jsp

```diff
@@ -141,19 +141,45 @@ Map<Long, List<CommerceOrderValidatorResult>> commerceOrderValidatorResultMap =
 					%>
 
 					<liferay-ui:search-container-column-text
-						name="list-price"
+						name="price"
 					>
 						<c:if test="<%= commerceCartContentDisplayContext.hasViewPricePermission() %>">
 
 							<%
 							CommerceMoney unitPriceCommerceMoney = commerceOrderItem.getUnitPriceMoney();
+							CommerceMoney unitPromoPriceCommerceMoney = commerceOrderItem.getPromoPriceMoney();
 
 							if (commercePriceDisplayType.equals(CommercePricingConstants.TAX_INCLUDED_IN_PRICE)) {
```

Comment:
@riccardo-alberti I really don't like those checkes int the view, we need to find better ways

Reply:
I will accept now to don't block, but please put in the backlog